CONTRIBUTING.md

```diff
@@ -10,11 +10,11 @@ Below you'll find information on how to setup a local development environment an
 
 In **[our Wiki](https://github.com/CMSgov/design-system/wiki)** you can find additional information like:
 
-- Pattern proposal process
-- Coding guidelines
-- Guiding principles
-- How to write documentation
-- etc.
+* Pattern proposal process
```

Comment:
Prettier 😞

Reply:
I didn't realize Prettier had an opinion about markdown. That's unfortunate